src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import org.junit.After;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private final FruitDao fruitDao = new FruitDaoImpl();
+
+    @After
+    public void tearDown() {
+        Storage.fruits.clear();
+    }
+
+    @Test
+    public void update_OneFruit_Ok() {
+        int fruitNumber = 33;
+        fruitDao.update("banana", fruitNumber);
+        int actual = Storage.fruits.get("banana");
+        assertEquals(fruitNumber, actual);
+    }
+
+    @Test
+    public void update_TwoFruits_Ok() {
+        fruitDao.update("banana", 13);
+        fruitDao.update("apple", 18);
+        int actualBanana = Storage.fruits.get("banana");
+        int actualApple = Storage.fruits.get("apple");
+        int expectedBanana = 13;
+        int expectedApple = 18;
+        assertEquals(expectedBanana, actualBanana);
+        assertEquals(expectedApple, actualApple);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void update_NullValue_NotOk() {
+        fruitDao.update("banana", null);
+        fruitDao.update(null, 6);
+        fruitDao.update(null, null);
```

Comment:
did program execution get to this lines (40, 41) if we get exception on line 39?